build-tools/packages/build-cli/src/commands/modify/fluid-imports.ts

```diff
@@ -147,6 +147,11 @@ async function updateImports(
 				}
 				const namedImports = importDeclaration.getNamedImports();
 
+				// Known issue: type-only import declarations lose the "type". Individual type imports retain theirs, but
+				// type-only ones are rewritten as regular imports, and type-only imports are not split out. Lint fixups should
+				// be able to restore the "type" if needed.
+				// const isTypeOnly = importDeclaration.isTypeOnly();
```

Comment:
This is a concern. We are way behind on getting type only imports fixed with a lint policy.
Type only impacts the API reports too. There are weird case, I think, where dropping a type only import will impact the resulting code (I think when a module has side-effects from load); so, I really don't want to drop type only imports. We must be able to preserve, them, yes.?

Reply:
Hmmm, I think I can preserve them by adding a flag to the import in the new import list. Not sure how to create a type-only import from scratch in TS-morph, but should be straightforward.

That said, I think we should prioritize getting our strict lint config enabled for all public packages with alpha or public exports. I think @Josmithr started that at one point; not sure how far he got.

Reply:
I was doing it as sort of an ad-hoc effort, not a tracked one. I believe the azure-client packages, odsp-client, fluid-static, and the public DDSs (aside from Tree, which @taylorsw04 is tracking) have been converted. Though there was at least one instance where a config was set back to minimal after I upgraded it :(